src/Microsoft.VisualStudio.Services.Agent/ConfigurationStore.cs

```diff
@@ -10,6 +10,22 @@
 
 namespace Microsoft.VisualStudio.Services.Agent
 {
+    public enum SignatureVerificationMode
```

Comment:
Nit: Order from least to most severe. And should we add Ints as the values?

Reply:
The ordering was selected so that Error is the default value.